sos/plugins/maas.py

```diff
@@ -53,18 +53,48 @@ def _remote_api_login(self):
         return ret['status'] == 0
 
     def setup(self):
-        self.add_copy_spec([
-            "/etc/squid-deb-proxy",
-            "/etc/maas",
-            "/var/lib/maas/dhcp*",
-            "/var/log/apache2*",
-            "/var/log/maas*",
-            "/var/log/upstart/maas-*",
-        ])
-        self.add_cmd_output([
-            "apt-cache policy maas-*",
-            "apt-cache policy python-django-*",
-        ])
+        snap_list_output = self.exec_cmd('snap list maas')
+        if snap_list_output["status"] == 0:
```

Comment:
I'd consider calling this something other then `_output`; historically names like `_out` and `_output` were common when we just returned command output, but today it's a dictionary of status and output data - I would suggest just calling it `snap_list` to avoid confusion.

Reply:
Renamed, thanks.